canonical/vanilla-framework
# Fixed theme of pin/close button in application layout examples [WD-2532]

## Done

Fixed theme of pin/close button in application layout examples.

Fixes [WD-2532](https://warthogs.atlassian.net/browse/WD-2532)

## QA

- Open an application layout example, e.g. the [default](https://vanilla-framework-4694.demos.haus/docs/examples/layouts/application/default) one.
- Zoom in, or resize the window, so that the side nav panel switches to the "drawer" mode.
- Check that the theme of the pin/close button is now consistent with that of the panel.

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.
  - if CSS class names are not changed it can be bugfix relesase (x.x.**X**)
  - if CSS class names are changed/added/removed it should be minor version (x.**X**.0)
  - see the [wiki for more details](https://github.com/canonical/vanilla-framework/wiki/Release-process#pre-release-tasks)

[WD-2532]: https://warthogs.atlassian.net/browse/WD-2532?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
